activerecord/test/activejob/query_comments_test.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require "activejob/helper"
+require "active_record/railties/query_comments"
+require "models/dashboard"
+
+class DashboardJob < ActiveJob::Base
+  def perform
+    Dashboard.first
+  end
+end
+
+ActiveJob::Base.include(ActiveRecord::Railties::QueryComments::ActiveJob)
+ActiveRecord::ConnectionAdapters::AbstractAdapter::QueryCommentContext.components = [:job]
```

Comment:
Same comment here as above, if these persist we need to move this to a setup/teardown block, otherwise state will leak between tests.